notebooks/experiments/conditional_diffusion/dna_diff_baseline_conditional_UNET.ipynb

```diff
@@ -0,0 +1,7119 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def one_hot_encode(seq, nucleotides, max_seq_len):
</pre><ul><li>Are the <code>seq</code> and <code>nucleotides</code> parameters, Lists or Strings ? The corresponding type hint should be added.</li><li>We should add the <code>int</code> type hint to <code>max_seq_len</code>.</li><li>We should add the <code>np.ndarray</code> type hint to the function output (viz. -&gt; <code>np.ndarray</code>)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinellolab/DNA-Diffusion/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>